.github/workflows/e2e-matrix.yml

```diff
@@ -112,6 +112,7 @@ jobs:
           - e2e-esbuild
           - e2e-eslint
           - e2e-expo
+          - e2e-gradle
```

Comment:
What about windows nightly? Do we not care about that one or it doesn't work?

Reply:
removed